.release/versions.hcl

```diff
@@ -6,12 +6,14 @@
 
 schema = 1
 active_versions {
+  version "1.19" {
+    ce_active = true
+  }
   version "1.18" {
     ce_active = true
     lts       = true
   }
   version "1.17" {}
-  version "1.16" {}
   version "1.15" {
     lts = true
```

Comment:
I know the docs say to change this AFTER the release, but it made more sense that starting now `backport/all` should be using 1.19. Or maybe we need to keep 1.16 so there is overlap in the interim?

Reply:
On second though, no more 1.18 releases are scheduled so this is fine.